bats/fb-test-puppet.bats

```diff
@@ -70,14 +70,6 @@ fi
   hammer puppet-environment update --name=production --locations "Default Location" --organizations "Default Organization"
 }
 
-@test "Assign proxy to default taxonomies" {
```

Comment:
Let's separate this out to its own PR. I think that can already be merged and reduces the scope of this PR.

Reply:
extracted to #1419